### ISSUE TITLE
Forgotten Realms: Demon Stone has multiple issues

### ISSUE THREAD

**Issue by CartoonFan** [Joshua-Ashton/d9vk#321]
Please describe your issue as accurately as possible. If you run into a problem with a binary release, make sure to test with latest `master` as well.

**Important:** When reporting an issue with a specific game or application, such as crashes or rendering issues, please include log files and a D3D11/D3D9 Apitrace (see https://github.com/apitrace/apitrace) so that the issue can be reproduced.
In order to create a trace for **D3D11/D3D10**: Run `wine apitrace.exe trace -a dxgi YOURGAME.exe`.
In order to create a trace for **D3D9**: Follow https://github.com/Joshua-Ashton/d9vk/wiki/Making-a-Trace.
DO NOT use DXVK together with apitrace!

### Software information
Forgotten Realms: Demon Stone

### System information
- GPU: Nvidia GeForce GTX 960
- Driver: 430.40.0
- Wine version:  4.14 TKG
- DXVK version: https://github.com/Joshua-Ashton/d9vk/commit/8490b27bf20b3ee20d0aa5d850e430841415e7e0

### Apitrace file(s)
Windows trace (AMD Graphics): https://drive.google.com/file/d/1SWVx7KwEV3KmGI_o8LD1rgKLrKOg5Y-2/view?usp=sharing

D9VK trace (Nvidia Graphics):
https://drive.google.com/file/d/1cVXHeDhA3tcf6mxyehAClSsLdZqUQSiM/view?usp=sharing

### Log files
- d3d9.log: [demonstone_d3d9.log](https://github.com/Joshua-Ashton/d9vk/files/3537509/demonstone_d3d9.log)
- d3d11.log: N/A
- dxgi.log: N/A

Hello! Demon Stone has several issues as of the current master. The HUD is missing some elements, models have white textures, and the loading screens aren't rendered properly.

D9VK:
![20190824-104333](https://user-images.githubusercontent.com/29664894/63641939-b4491700-c66b-11e9-8406-3afbc30d6920.png)
![20190824-104407](https://user-images.githubusercontent.com/29664894/63641946-d3e03f80-c66b-11e9-9b57-15546f1f7096.png)
![20190824-104338](https://user-images.githubusercontent.com/29664894/63641951-de023e00-c66b-11e9-9fd5-877500f87181.png)

Native (taken from Windows trace):
![20190824-104900](https://user-images.githubusercontent.com/29664894/63641958-012ced80-c66c-11e9-8c81-4547aad5c7dc.png)
![20190824-104922](https://user-images.githubusercontent.com/29664894/63641959-0f7b0980-c66c-11e9-842c-f4097dec3fd7.png)
![20190824-104913](https://user-images.githubusercontent.com/29664894/63641964-16a21780-c66c-11e9-91ef-6dc40b71dd1d.png)

The Windows trace plays back without errors on my Nvidia machine, so it might be an nvidia issue :confused:

Also, the character models occasionally freeze during animations(which isn't shown in either trace), and @NerosTie mentioned an issue with the logo (not sure if I covered that here already or not). Thanks!

**Comment by Joshua-Ashton** [comment]
Are you using Wine's d3dcompiler impl?

ie. did you install d3dcompiler[_43/_47/_whatever] and d3d9x stuff via proton/winetricks

**Comment by CartoonFan** [comment]
It doesn't seem like I'm using Wine's d3dcompiler. I installed d9vk through winetricks, and winetricks shows that d3dx9 and d3dcompiler_43 are installed. All those dlls are set to native from what I can see, and I went ahead and installed the extra d3dcompiler stuff just in case, but it looks like the issues persist.

**Comment by NerosTie** [comment]
The vertical red line is gone, but it just a black screen now (tested with dxvk 1101)

**Comment by CartoonFan** [comment]
The HUD objects are now covered with a black square, and the "all-white textures" and " intro walk animation halting" problems still persist, as well:
![forgotten_realms_20200128-192125](https://user-images.githubusercontent.com/29664894/73325689-47324280-4204-11ea-9f98-4ae792f3e8fd.png)
![forgotten_realms_20200128-192237](https://user-images.githubusercontent.com/29664894/73325713-5fa25d00-4204-11ea-91fd-5d18cf850165.png)
![forgotten_realms_20200128-192251](https://user-images.githubusercontent.com/29664894/73325718-64671100-4204-11ea-8095-9ce7ab8956b0.png)

**Comment by CartoonFan** [comment]
With https://github.com/doitsujin/dxvk/issues/1409 and https://github.com/doitsujin/dxvk/commit/b9474cf787f52e3faf8fa670bdb85ce058d47ade , everything except the character halting has been fixed. Good job!

**Comment by misyltoad** [comment]
Can you show me what that issue is?

**Comment by CartoonFan** [comment]
I apologize for my earlier statement. I honestly don't know what happened or how I came to my previous conclusion, but it seems pretty smooth now. When playing, there is some noticeable micro-stutter (am I using that correctly?) right after the camera turns to face the protagonist, however it's much harder to notice compared to before. It also didn't show up in my attempt to record the issue with OBS :ghost: , so I don't really know how to show it outside of describing it. My not-very-helpful failed attempt is [here](https://drive.google.com/file/d/14-y4e4p_RkC1uLlymfqV4XQ47IuM_RXV/view?usp=sharing) if you're curious (the relevant part would be at 0:12). I guess I'm OK closing it here if @NerosTie doesn't have any further objections.

Closing #321 for now.
